diff -ur -x .svn -x .git -x .bzr -x CVS -ur opensips-2.4.0.orig/Makefile opensips-2.4.0/Makefile
--- opensips-2.4.0.orig/Makefile	2018-04-30 21:37:06.000000000 +0900
+++ opensips-2.4.0/Makefile	2018-05-23 19:45:30.853360012 +0900
@@ -49,7 +49,6 @@
 # default debian version when running 'make deb'
 DEBIAN_VERSION ?= jessie #TODO: can we determine this?
 
-makefile_defs=0
 DEFS:= $(DEFS_EXTRA_OPTS)
 DEBUG_PARSER?=
 
@@ -467,7 +466,7 @@
 # Install everything (except modules-docbook?)
 install: install-app install-console install-modules-all
 
-opensipsmc: $(cfg_prefix)/$(cfg_dir) $(data_prefix)/$(data_dir)
+opensipsmc: $(cfg_dir) $(data_dir)
 	$(MAKE) -C menuconfig proper
 	$(MAKE) -C menuconfig \
 		MENUCONFIG_CFG_PATH=$(data_target)/menuconfig_templates/ \
@@ -484,14 +483,14 @@
 	$(MAKE) -C db/schema
 	-@echo "Done"
 
-mk-install-dirs: $(cfg_prefix)/$(cfg_dir) $(bin_prefix)/$(bin_dir) \
-			$(modules_prefix)/$(modules_dir) $(doc_prefix)/$(doc_dir) \
-			$(man_prefix)/$(man_dir)/man8 $(man_prefix)/$(man_dir)/man5 \
-			$(data_prefix)/$(data_dir)
+mk-install-dirs: $(cfg_dir) $(bin_dir) \
+			$(modules_dir) $(doc_dir) \
+			$(man_dir)/man8 $(man_dir)/man5 \
+			$(data_dir)
 
 
 # note: on solaris 8 sed: ? or \(...\)* (a.s.o) do not work
-install-cfg: $(cfg_prefix)/$(cfg_dir)
+install-cfg: $(cfg_dir)
 		sed -e "s#/usr/.*lib/$(NAME)/modules/#$(modules_target)#g" \
 			< etc/$(NAME).cfg > $(cfg_prefix)/$(cfg_dir)$(NAME).cfg.sample0
 		umask 0077; sed -e "s#/usr/.*etc/$(NAME)/tls/#$(cfg_target)tls/#g" \
@@ -500,7 +500,7 @@
 				$(cfg_prefix)/$(cfg_dir)/osipsconsolerc; \
 		fi
 
-install-console: $(bin_prefix)/$(bin_dir)
+install-console: $(bin_dir)
 		# install osipsconsole
 		cat scripts/osipsconsole | \
 		sed -e "s#PATH_BIN[ \t]*=[ \t]*\"\./\"#PATH_BIN = \"$(bin-target)\"#g" | \
@@ -529,7 +528,7 @@
 		$(INSTALL_BIN) /tmp/osipsconsole $(bin_prefix)/$(bin_dir)
 		rm -fr /tmp/osipsconsole
 
-install-bin: $(bin_prefix)/$(bin_dir) opensipsmc utils
+install-bin: $(bin_dir) opensipsmc utils
 		# install opensips binary
 		$(INSTALL_TOUCH) $(bin_prefix)/$(bin_dir)/$(NAME)
 		$(INSTALL_BIN) $(NAME) $(bin_prefix)/$(bin_dir)
@@ -604,7 +603,7 @@
 			cd utils/db_oracle; $(MAKE) all ; \
 		fi ;
 
-install-modules: modules $(modules_prefix)/$(modules_dir)
+install-modules: modules $(modules_dir)
 	@for r in $(modules_full_path) "" ; do \
 		if [ -n "$$r" ]; then \
 			if [ -f "$$r" ]; then \
@@ -622,7 +621,7 @@
 .PHONY: install-doc install-app-doc install-modules-doc
 install-doc: install-app-doc install-modules-doc
 
-install-app-doc: $(doc_prefix)/$(doc_dir)
+install-app-doc: $(doc_dir)
 	-@for d in $(install_docs) ""; do \
 		if [ -n "$$d" ]; then \
 			$(INSTALL_TOUCH) $(doc_prefix)/$(doc_dir)/"$$d" ; \
@@ -632,7 +631,7 @@
 	done
 
 
-install-modules-doc: $(doc_prefix)/$(doc_dir)
+install-modules-doc: $(doc_dir)
 	-@for r in $(modules_basenames) "" ; do \
 		if [ -n "$$r" ]; then \
 			if [ -f modules/"$$r"/README ]; then \
@@ -642,7 +641,7 @@
 	done
 
 
-install-man: $(man_prefix)/$(man_dir)/man8 $(man_prefix)/$(man_dir)/man5
+install-man: $(man_dir)/man8 $(man_dir)/man5
 		sed -e "s#/etc/$(NAME)/$(NAME)\.cfg#$(cfg_target)$(NAME).cfg#g" \
 			-e "s#/usr/sbin/#$(bin-target)#g" \
 			-e "s#/usr/lib/$(NAME)/modules/#$(modules_target)#g" \
@@ -652,7 +651,7 @@
 			$(man_prefix)/$(man_dir)/man8/opensipsunix.8
 		chmod 644  $(man_prefix)/$(man_dir)/man8/opensipsunix.8
 
-install-modules-docbook: $(doc_prefix)/$(doc_dir)
+install-modules-docbook: $(doc_dir)
 	-@for r in $(modules_basenames) "" ; do \
 		if [ -n "$$r" ]; then \
 			if [ -d modules/"$$r"/doc ]; then \
diff -ur -x .svn -x .git -x .bzr -x CVS -ur opensips-2.4.0.orig/Makefile.defs opensips-2.4.0/Makefile.defs
--- opensips-2.4.0.orig/Makefile.defs	2018-04-30 21:37:06.000000000 +0900
+++ opensips-2.4.0/Makefile.defs	2018-05-23 19:46:05.110927079 +0900
@@ -52,11 +52,6 @@
 
 # check if already included/exported
 
-ifeq ($(makefile_defs), 1)
-else
-makefile_defs=1
-export makefile_defs
-
 # main binary name
 MAIN_NAME=opensips
 
@@ -1657,5 +1652,3 @@
 test_objs := $(test_src:.c=.o)
 objs := $(objs) $(test_objs)
 endif
-
-endif  # ifeq ($(makefile_defs, 1)
diff -ur -x .svn -x .git -x .bzr -x CVS -ur opensips-2.4.0.orig/Makefile.rules opensips-2.4.0/Makefile.rules
--- opensips-2.4.0.orig/Makefile.rules	2018-04-30 21:37:06.000000000 +0900
+++ opensips-2.4.0/Makefile.rules	2018-05-23 19:46:45.491993230 +0900
@@ -92,26 +92,26 @@
 		END { print "#endif /* __MEM_STAT_H_ */\n#endif /* SHM_EXTRA_STATS */"; \
 		print "/* modules MD5: $(MODULES_MD5) */"; fflush()}' > $@
 
-$(cfg_prefix)/$(cfg_dir):
+$(cfg_dir):
 		mkdir -m 700 -p $(cfg_prefix)/$(cfg_dir)
 
-$(bin_prefix)/$(bin_dir):
+$(bin_dir):
 		mkdir -p $(bin_prefix)/$(bin_dir)
 
-$(modules_prefix)/$(modules_dir):
+$(modules_dir):
 		mkdir -p $(modules_prefix)/$(modules_dir)
 
-$(doc_prefix)/$(doc_dir):
+$(doc_dir):
 		mkdir -p $(doc_prefix)/$(doc_dir)
 
-$(man_prefix)/$(man_dir)/man8:
+$(man_dir)/man8:
 		mkdir -p $(man_prefix)/$(man_dir)/man8
 
-$(man_prefix)/$(man_dir)/man5:
+$(man_dir)/man5:
 		mkdir -p $(man_prefix)/$(man_dir)/man5
 
-$(data_prefix)/$(data_dir):
+$(data_dir):
 		mkdir -p $(data_prefix)/$(data_dir)
 
 
 .PHONY: dosetrev
