diff -up opensips-1.6.0/tls/tls_init.c.openssl opensips-1.6.0/tls/tls_init.c
--- opensips-1.6.0/tls/tls_init.c.openssl	2008-11-24 21:52:33.000000000 +0100
+++ opensips-1.6.0/tls/tls_init.c	2009-08-25 20:18:45.000000000 +0200
@@ -468,8 +468,8 @@ static int check_for_krb(void)
 	if (xx==NULL)
 		return -1;

-	for( j=0 ; j<M_sk_num(xx->cipher_list) ; j++) {
-		SSL_CIPHER *yy = (SSL_CIPHER*)M_sk_value(xx->cipher_list,j);
+	for( j=0 ; j<sk_SSL_CIPHER_num(xx->cipher_list) ; j++) {
+		SSL_CIPHER *yy = sk_SSL_CIPHER_value(xx->cipher_list,j);
 		if ( yy->id>=SSL3_CK_KRB5_DES_64_CBC_SHA &&
 		 yy->id<=SSL3_CK_KRB5_RC4_40_MD5 ) {

